lib/krb5/test_acl.c

```diff
@@ -37,7 +37,7 @@
 	do { if (r != e) krb5_errx(c, 1, "%s", s); } while (0)
 #define STRINGMATCH(c, s, _s1, _s2) \
 	do {							\
-		if (_s1 == NULL || _s2 == NULL) 		\
+		if (_s1 == NULL)				\
```

Comment:
Exclude this fix.

Reply:
ok, but what was it supposed to be fixing?

Reply:
This macro is only ever used with a constant string literal as `_s2`, so VC complains that that comparison to `NULL` is pointless.

Reply:
Thanks for the explanation